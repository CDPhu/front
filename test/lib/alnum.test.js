const alnum = require("../../lib/alnum")

describe("alnum.isAlphaNumeric", () => {
    test("0", () => {
        let result = alnum.isAlphaNumeric({ length: 0, charCodeAt: () => 143 })
        expect(result).toMatchSnapshot()
    })

    test("1", () => {
        let result = alnum.isAlphaNumeric({ length: 16, charCodeAt: () => 176 })
        expect(result).toMatchSnapshot()
    })

    test("2", () => {
        let result = alnum.isAlphaNumeric({ length: 64, charCodeAt: () => 224 })
        expect(result).toMatchSnapshot()
    })

    test("3", () => {
        let result = alnum.isAlphaNumeric({ length: 10, charCodeAt: () => 239 })
        expect(result).toMatchSnapshot()
    })

    test("4", () => {
        let result = alnum.isAlphaNumeric({ length: 32, charCodeAt: () => 135 })
        expect(result).toMatchSnapshot()
    })

    test("5", () => {
        let result = alnum.isAlphaNumeric({ length: Infinity, charCodeAt: () => Infinity })
        expect(result).toMatchSnapshot()
    })
})
