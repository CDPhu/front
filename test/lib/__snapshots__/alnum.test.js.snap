// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alnum.isAlphaNumeric 0 1`] = `true`;

exports[`alnum.isAlphaNumeric 1 1`] = `false`;

exports[`alnum.isAlphaNumeric 2 1`] = `false`;

exports[`alnum.isAlphaNumeric 3 1`] = `false`;

exports[`alnum.isAlphaNumeric 4 1`] = `false`;

exports[`alnum.isAlphaNumeric 5 1`] = `false`;
